src/monitoring/ilm.js

```diff
@@ -0,0 +1,3 @@
+export const ILM_OUTPUT_PROP = 'ILM Output';
```

Comment:
@paulschreiber I think so, this makes sense to me since this values are related directly to the ILM outputs, these are and will be used to send a propery to plausible with the specific ILM Output relevant to that metric, so in here we can define the constants that if the specific values change over time we can handle just in one place